crates/cheatcodes/src/inspector.rs

```diff
@@ -647,6 +647,7 @@ impl<DB: DatabaseExt + Send> Inspector<DB> for Cheatcodes {
         }
     }
 
+    #[inline]
```

Comment:
I guess this is to mirror https://github.com/foundry-rs/foundry/pull/8197?

Reply:
Yeah the `step_end` hook wasn't being called properly so I mirrored it exactly as on upstream.